python/tests/test_snippets.py

```diff
@@ -82,6 +82,13 @@ def test_window_snippet(house: etree.ElementTree) -> None:
     }
 
 
+def test_heated_floor_area_snippet(house: etree.ElementTree) -> None:
```

Comment:
`house` is actually `etree._Element`. My PR (#109)  updates the rest of these to that input type.